src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GithubWebTests.kt

```diff
@@ -0,0 +1,42 @@
+package org.vaccineimpact.orderlyweb.customConfigTests
+
+import khttp.get
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.vaccineimpact.orderlyweb.db.AppConfig
+
+
+class GithubWebTests : CustomConfigTests()
```

Comment:
I struggled a bit to think of integration tests I could do here. We can spoof an access token, but the token is provided in a response from Github in the OAuth web flow and converted into a session cookie. So it's tricky to test the second part of the login process..

Reply:
Added a new ticket for Selenium tests: https://vimc.myjetbrains.com/youtrack/issue/mrc-218